SoftLayer/CLI/virt/detail.py

```diff
@@ -97,12 +97,10 @@ def cli(env, identifier, passwords=False, price=False):
         total_price = utils.lookup(result,
                                    'billingItem',
                                    'nextInvoiceTotalRecurringAmount') or 0
-        total_price += sum(p['nextInvoiceTotalRecurringAmount']
-                           for p
-                           in utils.lookup(result,
-                                           'billingItem',
-                                           'children') or [])
-        table.add_row(['price_rate', total_price])
+        if total_price != 0:
+            price_data = _get_price_data(utils.lookup(result, 'billingItem'), total_price)
+            table.add_row(['Prices', price_data['priceTable']])
+            table.add_row(['Price rate', price_data['totalPrice']])
```

Comment:
This might have just been a bug with the old implementation, but I think `nextInvoiceTotalRecurringAmount` should equal the sum of all child billing items. The `total` in the prices table should equal this amount.

```suggestion
            table.add_row(['Price rate', total_price  ])
```

Reply:
the command was updated to match `nextInvoiceTotalRecurringAmount` and `total`
![image](https://user-images.githubusercontent.com/78806346/198642606-82aadcb8-cb81-4723-a9da-f45003181af3.png)